appsignals-tests/contract-tests/src/test/java/software/amazon/opentelemetry/appsignals/test/jdbc/JdbcH2Test.java

```diff
@@ -31,13 +31,14 @@ public class JdbcH2Test extends JdbcContractTestBase {
   private static final String DB_PLATFORM = "org.hibernate.dialect.H2Dialect";
 
   @Test
-  public void testSuccess() {
-    assertSuccess(DB_SYSTEM, DB_OPERATION, DB_USER, DB_NAME, DB_CONNECTION_STRING, null, null);
+  public void testSuccessSelect() {
```

Comment:
I added for MySQL and Postgres which are needed for the contract tests here.

I tried if H2 was needed but I can see not needed and additionally I tried to go further and understand how DBs are created on H2 since I don't see for it SQL grammar here: https://www.h2database.com/html/grammar.html

So going further from there it seems like a new DB is created by changing the connection URL https://www.h2database.com/html/tutorial.html#creating_new_databases

So for now not adding this contract test to H2, I tried to see the result but got Bad SQL Grammar Exception.

Reply:
Ok so it looks like H2 doesn't easily support creation of new databases, that's fine.